pkgs/applications/misc/nut/default.nix

```diff
@@ -1,5 +1,5 @@
-{ lib, stdenv, fetchurl, substituteAll, pkg-config, neon, libusb-compat-0_1, openssl, udev, avahi, freeipmi
-, libtool, makeWrapper, autoreconfHook, fetchpatch
+{ lib, stdenv, fetchurl, substituteAll, pkg-config, neon, libusb, openssl, udev, avahi, freeipmi
```

Comment:
This should be `libusb1` (`libusb` does not exist)?

Reply:
`libusb` seems to be an alias of `libusb1` (https://github.com/NixOS/nixpkgs/blob/8f13c7d6d718e8aa25fef5ba00e901f9c2d6b48f/pkgs/top-level/aliases.nix#L899), but you're right, we should be consistent with every other package that uses libusb1.

Reply:
OfBorg was complaining about `libusb`, but it could be that OfBorg tries to enforce cleanups with respect to aliases (?).